tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -282,9 +267,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             fixtures: {
                 type: "fluid.test.testCaseHolder",
                 options: {
```

Comment:
This should all be in a separate grade that uses `fluid.test.testCaseHolder` as a gradeName and then passed in here as the type.